runtime-sdk/src/testing/mock.rs

```diff
@@ -25,32 +37,32 @@ pub struct Mock {
     pub consensus_state: ConsensusState,
     pub epoch: beacon::EpochTime,
 
-    pub methods: MethodRegistry,
-
     pub max_messages: u32,
 }
 
 impl Mock {
     /// Create a new mock dispatch context.
-    pub fn create_ctx(&mut self) -> DispatchContext<'_> {
-        DispatchContext {
-            mode: Mode::ExecuteTx,
-            runtime_header: &self.runtime_header,
-            runtime_round_results: &self.runtime_round_results,
-            runtime_storage: storage::MKVSStore::new(
-                IoContext::background().freeze(),
-                self.mkvs.as_mut(),
-            ),
-            consensus_state: &self.consensus_state,
-            epoch: self.epoch,
-            io_ctx: IoContext::background().freeze(),
-            methods: &self.methods,
-            logger: get_logger("mock"),
-            block_tags: Tags::new(),
-            messages: Vec::new(),
-            max_messages: self.max_messages,
-            values: BTreeMap::new(),
-        }
+    pub fn create_ctx(
```

Comment:
:information_source: existing uses of this don't re-dispatch anything, so they can use this empty-runtime context without any changes